src/components/EditableHeading/EditableHeading.jsx

```diff
@@ -200,8 +198,8 @@ const EditableHeading = props => {
   );
 };
 
-EditableHeading.types = TYPES;
-EditableHeading.sizes = SIZES;
+EditableHeading.types = Heading.types;
```

Comment:
Let's switch to object assign format as we did in other components

Reply:
This component haven't been converted to TS yet - will do it while converting